docs/Contributing/FAQ.md

```diff
@@ -5,6 +5,8 @@
   - [`undefined: Asset`](#undefined-asset)
 - [How do I connect to the MailHog simulated mail server?](#how-do-i-connect-to-the-mailhog-simulated-mail-server)
 - [Adding hosts for testing](#adding-hosts-for-testing)
+- [Why am I getting an error about self-signed certificates when running `fleetctl preview`?](#why-am-i-getting-an-error-about-self-signed-certificates-when-running-fleetctl-preview)
```

Comment:
Why is this on the "contributing" FAQ page? Not "using"?

Reply:
Good question. This is because, when Fleet sandbox is released, `fleetctl preview will transition to a tool used by Fleet contributors to test changes to Fleet. 

Fleet sandbox becomes the recommended way to try Fleet.